### ISSUE TITLE
[3.4.0-alpha-01] nested .eslintignore file ignored

### ISSUE THREAD

**Issue by Edelf** [react-workspaces/react-workspaces-playground#47]
After upgrading to 3.4.0-alpha-01 I started to have lint errors in serviceWorker.ts even tho the file is listed in the .eslintignore:

```
Failed to compile.

./packages/apps/my-ts-app/src/serviceWorker.ts
  Line 47:9:  'checkValidServiceWorker' was used before it was defined  @typescript-eslint/no-use-before-define
  Line 59:9:  'registerValidSW' was used before it was defined          @typescript-eslint/no-use-before-define

Search for the keywords to learn more about each error.
```

<root>/packages/apps/.eslintignore:
```
**/node_modules/**
**/dist/**
**/storybook-static/**
**/coverage/**
**/build/**
**/.git/**
**/public/**
serviceWorker.ts
```

The only way for serviceWorker.ts to be ignored is to add it to the .eslintignore at the root:
<root>.eslintignore:
```
**/node_modules/**
**/dist/**
**/storybook-static/**
**/coverage/**
**/build/**
**/.git/**
**/public/**
!.eslintrc.js
serviceWorker.ts
```

Steps to reproduce:
- checkout this repo
- remove non-TS projects (leaving only components-typescript and apps\app-typescript )
- yarn install (using 1.19.0)
- yarn upgrade-interactive --latest  (https://i.imgur.com/kxBhULx.png)
- yarn workspace @project/app-typescript start

**Comment by alanrubin** [comment]
True, that happens to me as well after upgrading to 3.4.0-alpha-01

**Comment by F1LT3R** [comment]
I notice you interactive upgrade updates `babel-eslint.` This may be a 'babel-eslint` issue. 

If you downgrade `babel-eslint` after the interactive upgrade, what happens?

**Comment by Edelf** [comment]
I didn't select it to update. Only selected ( * ) the @react-workspaces dependency
